lib/src/base-styles/_nx-tile.scss

```diff
@@ -117,6 +126,7 @@ $nx-tile-background-color: $nx-white;
     .nx-tile-header__description {
```

Comment:
What is this class for? I don't see it demonstrated on the nx-tile page

Reply:
> What is this class for? I don't see it demonstrated on the nx-tile page

It's a holdover from IQ and like the title only tile I think it should be removed.

Reply:
> > What is this class for? I don't see it demonstrated on the nx-tile page
> 
> It's a holdover from IQ and like the title only tile I think it should be removed.

Which I have done here: https://github.com/sonatype/sonatype-react-shared-components/pull/34/commits/d7fc4b491b94534ea2784871f82a8374bb6897af